R/type.R

```diff
@@ -67,16 +75,7 @@ xportr_type <- function(.df, metacore, domain = NULL,
     data.frame(variable = names(.df), type = unlist(table_cols_types)),
     metacore,
     by = "variable"
-  ) %>%
-    mutate(
-      # _character is used here as a mask of character, in case someone doesn't
-      # want 'character' coerced to character
-      type.x = if_else(type.x %in% characterTypes, "_character", type.x),
-      type.x = if_else(type.x %in% numericTypes, "_numeric", type.x),
-      type.y = tolower(type.y),
-      type.y = if_else(type.y %in% characterTypes, "_character", type.y),
-      type.y = if_else(type.y %in% numericTypes, "_numeric", type.y)
-    )
+  )
```

Comment:
@EeethB Were these changes causing an issue? Or was it just a merge that took these out?

Reply:
I think that was just a merge, sorry about that. I will put them back in